### PR TITLE
Improve multi-region schema change docs

### PR DESCRIPTION
**Pull request by rmloveland**
Fixes DOC-8346

### CODE REVIEW

File: src/current/_includes/v22.2/performance/lease-preference-system-database.md [hunk 1]
```diff
@@ -7,4 +7,9 @@ ALTER DATABASE system CONFIGURE ZONE USING constraints = '{"+region=us-east1": 1
 
 Run all subsequent schema changes from a node in the specified region.
 
-If you do not intend to run more schema changes from that region, you can safely remove the lease preference from the zone configuration for the system database.
+If you do not intend to run more schema changes from that region, you can safely remove the lease preference from the zone configuration for the system database using the following [`ALTER DATABASE ... CONFIGURE ZONE`]({% link {{ page.version.version }}/alter-database.md %}#configure-zone) statement:
```
Comment (fqazi) [inline src/current/_includes/v22.2/performance/lease-preference-system-database.md:10]: See my comment on the JIRA, but we probably want to encourage people to stick to a region for schema changes. In general having a fixed lease preference is also more deterministic for other subsystems.

Comment (rmloveland) [reply src/current/_includes/v22.2/performance/lease-preference-system-database.md:10]: OK so based on your comment I'm inclined to actually _remove_ the stuff about dropping the zone config 

Specifically, I'd like to delete everything in this note after the line

> Run all subsequent schema changes from a node in the specified region.

What do you think of doing that?  That way most users will not keep touching this after they set it. But experts will still continue to have access to the [`ALTER DATABASE ... CONFIGURE ZONE`](https://www.cockroachlabs.com/docs/v23.1/alter-database#configure-zone) docs and can do what they want

Comment (rmloveland) [reply src/current/_includes/v22.2/performance/lease-preference-system-database.md:10]: ok, pausing here as the Jira convo is ongoing

Comment (rmloveland) [reply src/current/_includes/v22.2/performance/lease-preference-system-database.md]: Updated the text of this PR to match the outcome of our discussion in the Jira issue. (Basically, just add a link.) PTAL!

**Review by fqazi — APPROVED** [review]

**Review by taroface — APPROVED** [review]
👍

**Pull request merged**